plugins/experimental/wasm/wasm_main.cc

```diff
@@ -28,6 +28,10 @@
 #include "include/proxy-wasm/wasmedge.h"
 #endif
 
+#ifdef WASMTIME
+#include "include/proxy-wasm/wasmtime.h"
```

Comment:
will this have any conflict with WARM above? Or will WASMTIME not be defined if WAMR is?

Reply:
Yup. That's controlled by Makefile.inc